tests/regress/expected/test_primary_failure.out

```diff
@@ -21,15 +21,21 @@ returns text as $$
 
     cmd = 'pg_ctl -l postmaster.log -D %s ' % datadir
     cmd = cmd + '-W -m %s %s' % (command_mode, command)
+    if 'plpython2u' == 'plpython2u':
+        return subprocess.check_output(cmd, stderr=subprocess.STDOUT, shell=True).replace('.', '')
+    else:
+        return subprocess.check_output(cmd, stderr=subprocess.STDOUT, shell=True, encoding='utf8').replace('.', '')
 
-    return subprocess.check_output(cmd, stderr=subprocess.STDOUT, shell=True).replace('.', '')
-$$ language plpythonu;
+$$ language plpython2u;
```

Comment:
seems, this file (at upstream) should be moved to a `in.out`, because `plpython2u` for gpdb 6, and `plpython3u` gpdb 7
(and this line is configured by cmake with @PLPYTHON_LANG_STR@ varable)

Reply:
I've missed the [init_file](https://github.com/arenadata/diskquota/pull/18/files#diff-e0364884734ecb435b1b113fc861c708db3150af18be60e1450f013f6cdf62b7R24), seems all ok